### PR TITLE
[TGER-59] Migrate Nova resource

### PR DESCRIPTION
**Pull request by phuclh**
- Bump dependencies to the latest version.
- Migrate Nova resource.

### CODE REVIEW

File: src/Nova/Block.php [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            nova('user') ? BelongsTo::make('Creator', 'creator', nova('user'))->exceptOnForms() : null,
+            DateTime::make('Created At')->exceptOnForms(),
+            DateTime::make('Updated At')->exceptOnForms(),
+        ]);
+    }
+
+    public function filters(Request $request)
+    {
+        return [
+            new Filters\FutureBlocks,
```
Comment (phuclh) [inline src/Nova/Block.php]: @drewroberts can you let me know what packages contain these filters so I can refactor them?

File: src/Nova/Game.php [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            DateTime::make('Created At')->exceptOnForms(),
+            nova('user') ? BelongsTo::make('Updated By', 'updater', nova('user'))->exceptOnForms() : null,
+            DateTime::make('Updated At')->exceptOnForms(),
+        ]);
+    }
+
+    public function filters(Request $request)
+    {
+        return [
+            new Filters\RoomLocation,
```
Comment (phuclh) [inline src/Nova/Game.php]: @drewroberts can you let me know what packages contain these filters so I can refactor them?

File: src/Nova/RecurringSchedule.php [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+            DateTime::make('Created At')->exceptOnForms(),
+            nova('user') ? BelongsTo::make('Updated By', 'updater', nova('user'))->exceptOnForms() : null,
+            DateTime::make('Updated At')->exceptOnForms(),
+        ]);
+    }
+
+    public function filters(Request $request)
+    {
+        return [
+            new Filters\RoomLocation,
```
Comment (phuclh) [inline src/Nova/RecurringSchedule.php]: @drewroberts can you let me know what packages contain these filters so I can refactor them?

File: src/Nova/ScheduleEraser.php [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+            nova('user') ? BelongsTo::make('Created By', 'creator', nova('user'))->exceptOnForms() : null,
+            DateTime::make('Created At')->exceptOnForms(),
+            DateTime::make('Updated At')->exceptOnForms(),
+        ]);
+    }
+
+    public function filters(Request $request)
+    {
+        return [
+            new Filters\RoomLocation,
```
Comment (phuclh) [inline src/Nova/ScheduleEraser.php]: @drewroberts can you let me know what packages contain these filters so I can refactor them?

File: src/Nova/Slot.php [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+            DateTime::make('Created At')->exceptOnForms(),
+            nova('user') ? BelongsTo::make('Updated By', 'updater', nova('user'))->exceptOnForms() : null,
+            DateTime::make('Updated At')->exceptOnForms(),
+        ]);
+    }
+
+    public function filters(Request $request)
+    {
+        return [
+            new Filters\FutureSlots,
```
Comment (phuclh) [inline src/Nova/Slot.php]: @drewroberts can you let me know what packages contain these filters so I can refactor them?

File: src/Nova/Game.php (hunk 2 above)
Comment (drewroberts) [reply src/Nova/Game.php]: Let's add this one to the escape room pacakge

File: src/Nova/Block.php (hunk 1 above)
Comment (drewroberts) [reply src/Nova/Block.php]: Let's add this one to this scheduling package

File: src/Nova/RecurringSchedule.php (hunk 3 above)
Comment (drewroberts) [reply src/Nova/RecurringSchedule.php]: https://github.com/tipoff/escape-room

File: src/Nova/ScheduleEraser.php (hunk 4 above)
Comment (drewroberts) [reply src/Nova/ScheduleEraser.php]: https://github.com/tipoff/escape-room

File: src/Nova/Slot.php (hunk 5 above)
Comment (drewroberts) [reply src/Nova/Slot.php]: This scheduling package

File: src/Nova/Block.php (hunk 1 above)
Comment (chrispecoraro) [reply src/Nova/Block.php]: I've added FutureBlocks.php.

Comment (drewroberts) [reply src/Nova/Block.php]: Thank you!

File: src/Nova/Game.php (hunk 2 above)
Comment (chrispecoraro) [reply src/Nova/Game.php]: I added this one, too.

**Comment by phuclh** [comment]
@chrispecoraro you moved GameFilters to Nova, that not a Nova filter.

**Review by drewroberts — APPROVED** [review]
Thank each of you! 🥞

**Pull request merged**